core/tests/protractor_utils/users.js

```diff
@@ -17,85 +17,51 @@
  * carrying out end-to-end testing with protractor.
  */
 
+var FirebaseAdmin = require('firebase-admin');
+var HashWasm = require('hash-wasm');
 var general = require('./general.js');
 var waitFor = require('./waitFor.js');
 var action = require('./action.js');
 var AdminPage = require('./AdminPage.js');
 var adminPage = new AdminPage.AdminPage();
 
-var _login = async function(
-    email, isSuperAdmin = false, manualNavigation = true) {
-  // Use of element and action is not possible because the login page
-  // is non-angular.
-  // The full url is also necessary.
-  var driver = browser.driver;
-  // The manualNavigation argument is used to determine whether to navigate to
-  // the login URL using driver.get() or not. If false, the calling method
-  // should handle navigation to the login page.
-  if (manualNavigation) {
-    await driver.get(general.SERVER_URL_PREFIX + general.LOGIN_URL_SUFFIX);
-  }
-  // The statement below uses a browser.wait() to determine if the user has
-  // logged in. Use of waitFor is not possible because the active page is
-  // non-angular.
-  await browser.wait(
-    async() => {
-      try {
-        await driver.findElement(protractor.By.name('email'));
-      } catch (error) {
-        return false;
-      }
-      return true;
-    }, waitFor.DEFAULT_WAIT_TIME_MSECS, 'Login takes too long.');
-  await (await driver.findElement(protractor.By.name('email'))).clear();
-  await (await driver.findElement(protractor.By.name('email'))).sendKeys(email);
+var _createFirebaseAccount = async function(email, isSuperAdmin = false) {
+  var user = await FirebaseAdmin.auth().createUser({
+    email: email,
+    emailVerified: true,
+    password: await HashWasm.md5(email),
```

Comment:
Is this just for testing? I think we should definitely not be using MD5 for anything security-related.